src/engine/engine.rs

```diff
@@ -51,6 +51,12 @@ pub trait Dev: Debug {
 pub trait Filesystem: HasName + HasUuid {
     /// Rename this filesystem.
     fn rename(&mut self, name: &str) -> ();
+
+    /// Get this filesystem's mount point
+    fn mountpoint(&self) -> &String;
+
+    /// Get this filesystem's quota
+    fn quota(&self) -> &Bytes;
```

Comment:
Maybe we should not have put these in the DBus API yet then. We should remove them there, and also here, for now.